### ISSUE TITLE
🐞 Bug Report : [v3.8-Beta] Sticky - stay in column jumps when placed on a nested container

### ISSUE THREAD

**Issue by breakerh** [elementor/elementor#19777]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When you put a container inside a container and set the child container to sticky with stay in column activated, as soon as it hits the bottom it jumps to the left because `inset-inline-start` isn't set to 0 (zero).

I made a quick and dirty fix that solves it.
1. give the sticky container the following class: `elementor-sticky--nested`
2. add the following custom css (or in a global style if needed)
```css
selector.elementor-sticky--active.elementor-sticky--fix {
    inset-inline-start: 0 !important;
}
```
3. paste the following code en elementor code (or in a html tag on page, but i don't recommend that!)
```html
<script defer>
	document.addEventListener("readystatechange",()=>{
	    if(document.readyState !== 'complete')
	        return;
		const esItems = document.querySelectorAll('.elementor-sticky--nested');
		if(esItems.length<1)
		    return;
		/* 
		 * esItems.forEach((stickyContainer) => {
		 *	console.log('init sticky tracker');
		 * });
		*/
		window.addEventListener('scroll',()=>{
		    esItems.forEach((stickyContainer)=>{
		        if(!stickyContainer.classList.contains('elementor-sticky--active'))
		            return;
		        if(stickyContainer.style.bottom === '0px' && !stickyContainer.classList.contains('elementor-sticky--fix'))
		            stickyContainer.classList.add('elementor-sticky--fix');
	                else if(stickyContainer.style.bottom === '' && stickyContainer.classList.contains('elementor-sticky--fix'))
	                    stickyContainer.classList.remove('elementor-sticky--fix');
		    });
		});
	});
</script>
```

That's it

### Steps to reproduce

1) place container inside another container
2) add some content to the container (which in my case is higher then the viewport height)
3) activate sticky and stay in column
4) scroll :) 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

https://pastebin.com/4crdSv3X

**Comment by sbtre** [comment]
Please fix this issue ASAP. We have multiple websites waiting for go-live - because of this. @rami-elementor @mark-elementor

**Comment by breakerh** [comment]
Sorry for the late reply, But happy to help and I hope my quick 'n dirty fix helps solving it in the next release :)

**Comment by arielk** [comment]
This issue has been resolved in Elementor v3.8.0-beta-6

Feel free to update

Thanks!
